### PR TITLE
test-case: add ability to loop xrun injection test

### PR DESCRIPTION
**Pull request by stripes416**
* Allows for test to be run multiple times per pcm as needed.
* Ensures matching test coverage during transition to Github sof-test

Signed-off-by: Kelly Ledford <kelly.ledford@intel.com>

### CODE REVIEW

File: test-case/check-xrun-injection.sh [hunk 1]
```diff
@@ -51,9 +55,9 @@ FILE['capture,both']='/dev/null'
 
 func_xrun_injection()
 {
-    count=1
-    while(true)
-    do
+    count=0
+
+    if [[ $count -lt $loop_count ]]; then
```
[the remark this answers is not in the document]
Comment (keqiaozhang) [reply test-case/check-xrun-injection.sh]: @aiChaoSONG 
I have submitted a patch to refine the logic of this part. Please refer to https://github.com/thesofproject/sof-test/pull/31 and https://github.com/thesofproject/sof-test/pull/32.

**Comment by xiulipan** [comment]
@stripes416 I see duplicated PR from @keqiaozhang #31 #32 
I will close this and let @keqiaozhang to finish his work

File: test-case/check-xrun-injection.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (stripes416) [reply test-case/check-xrun-injection.sh]: whoops. good catch. will fix

**Comment by stripes416** [comment]
> @stripes416 I see duplicated PR from @keqiaozhang #31 #32
> I will close this and let @keqiaozhang to finish his work

ahh --- i didn't see those yesterday before i pushed. Thanks @xiulipan

**Pull request closed without merging**